recipes/ittapi/all/conanfile.py

```diff
@@ -0,0 +1,71 @@
+from conan import ConanFile
+from conan.tools.cmake import CMake, CMakeToolchain, cmake_layout
+from conan.tools.files import copy, get
+import os
+
+required_conan_version = ">=1.47.0"
+
+
+class IttApiConan(ConanFile):
+    name = "ittapi"
+    license = "dual licensed under GPLv2 and 3-Clause BSD licenses"
```

Comment:
```suggestion
    license = ("BSD-3-Clause", "GPL-2.0-only")
```
Only SPDX short names should be used, so we can filter 
https://spdx.org/licenses/